src/inc/quic_platform_winkernel.h

```diff
@@ -955,6 +955,8 @@ CxPlatRandom(
 #define QUIC_SILO PESILO
 #define QUIC_SILO_INVALID ((PESILO)(void*)(LONG_PTR)-1)
 
+#define QuicSiloGetHostServer() (PsGetHostSilo())
```

Comment:
While Windows refers to these silos as "Server Containers" or "Server Silos", the use of the word "Server" at the end of the names here is a bit confusing because it seems to imply a "client / server" type situation.

`QuicSiloGetHostSilo`, `QuicSiloIsInSilo`, `QuicSiloGetCurrentSilo` might make more sense.

Reply:
I agree. I followed the existing bad naming pattern, `QuicSiloGetCurrentServer`. I will update all of them.